bidwire/document.py

```diff
@@ -12,15 +12,17 @@ class Site(Enum):
         # TODO(anaulin): Remove user-visible site string representation from enum.
         MASSGOV_EOPSS = "the EOPSS pages"
         MEMPHIS_COUNCIL_CALENDAR = "the Memphis City Council Meeting Calendar"
+        CITYOFBOSTON = "Boston public notices"
```

Comment:
Let's not call this `CITYOFBOSTON`, as it is confusing with the existing scraper that looks at cityofboston.gov. Possible alternatives: `BOSTON_NOTICES`, `BOSTON_PUBLIC_NOTICES`, `BOSTON_PUB_NOTICES`.